README.md

```diff
@@ -221,13 +221,20 @@ Can be used to compare the versions, for instance when implementing version pred
 ### Example
 
 ```scala
+import Version._
```

Comment:
Kudos for including the import in the readme!

I was about to suggest that when I was first trying to use `Version` and was wondering where the string interpolator `semver` should come from.